src/clj/game/cards/upgrades.clj

```diff
@@ -900,10 +900,7 @@
                               (reveal state side target)
                               (shuffle! state side :deck)
                               (move state side target :hand)
-                              (effect-completed state side eid)))}
-              :no-ability
-              {:effect (effect (system-msg "declines to use Malapert Data Vault")
-                               (effect-completed eid))}}}]})
```

Comment:
Why do we want not to offer this feature?

Reply:
oops, I removed too much :joy: